setup.cfg

```diff
@@ -0,0 +1,88 @@
+[pycodestyle]
+max_line_length = 160
```

Comment:
I assume these are consistent with what we have in AG?

Reply:
Yes, this is the length in tabular